### PR TITLE
backend-webpack-tools for release Replace clientId

### PR DESCRIPTION
**Pull request by williamkbentley**

### CODE REVIEW

**Comment by williamkbentley** [comment]
@calebmshafer @wgoehrig Can you review? Current release of this app cannot be launched.

File: .github/workflows/create_release.yml [hunk 1]
```diff
@@ -20,7 +20,13 @@ jobs:
           git config --global user.name "imodeljs-admin[bot]"
         name: git config
       - run: npm ci
-      - run: npx run-s lint build build:dist
+      - run: npx run-s lint build build:backend:webpack
+      - uses: actions/github-script@v2
```
Comment (wgoehrig) [inline .github/workflows/create_release.yml:24]: Does this action actually use powershell?  The [docs for it](https://github.com/marketplace/actions/github-script) make it seem like it's supposed to be JS...

Comment (williamkbentley) [reply .github/workflows/create_release.yml]: You're right. I was looking at the create release step, didnt realize it was JS

**Review by wgoehrig — APPROVED** [review]

File: tsconfig.backend.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
   "exclude": [
     "build",
     "lib",
-    "node_modules"
+    "node_modules",
+    "scripts"
```
Comment (calebmshafer) [inline tsconfig.backend.json:16]: probably don't need this anymore or the one in the `tsconfig.json`

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**